src/user-data.ts

```diff
@@ -162,6 +162,11 @@ export type AppBottomBarCourse = {
   readonly uniqueID: number;
 };
 
+export type ShowAllCourses = {
+  readonly name: string;
+  readonly courses: AppCourse[];
+};
+
```

Comment:
I don't think this type should be in user-data. User data only contains the type of the data in firestore or its converted form. This type seems to be frontend display-only.

Reply:
O I see that makes sense